tslint.json

```diff
@@ -38,6 +38,10 @@
     },
     "member-access": {
       "severity": "none"
-    }
+    },
+    "variable-name": {
+      "options": ["allow-leading-underscore", "allow-pascal-case"]
+    },
+    "object-literal-key-quotes": [true, "as-needed"]
```

Comment:
Just confirming @ActuallyACat if these updates are alright (I noticed you added `tslint:latest` to the config). It allows for the `_` prefix convention for private methods (pending confirmation in #pract_front_end_eng if this is the way to go), and to only require `"` marks around object keys when needed.

Reply:
Yup, I'm okay with this. 👍